src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -170,13 +172,27 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
 
         if (isThreeDSecureRequiredError) {
             const clientSecret = error.body.three_ds_result.token;
-            const result = await this._getStripeJs().handleCardAction(clientSecret);
+            let result;
+            let needsConfirm = false;
+
+            if (this._reusePaymentIntent) {
+                try {
+                    result = await this._getStripeJs().handleCardAction(clientSecret);
+                    needsConfirm = true;
+                } catch (errorCardAction) {
```

Comment:
If `this._reusePaymentIntent` being `true` represents the BE working in the 'new flow', why do we need this `catch`? 

I was expecting something like:


```
            if (this._reusePaymentIntent) {
                // new flow...
                result = await this._getStripeJs().confirmCardPayment(clientSecret);
            } else {
                // old flow...
                result = await this._getStripeJs().handleCardAction(clientSecret);
                needsConfirm = true;
            }
```

Reply:
@leeBigCommerce I made the changes, I have some problems with the test after a rebase, but it is ready